reporter/src/main/kotlin/reporters/ReportTableModel.kt

```diff
@@ -214,6 +214,6 @@ data class ReportTableModel(
             val source: String,
             val description: String,
             val resolutionDescription: String,
-            val isResolved: Boolean
+            val isAddressed: Boolean
```

Comment:
Just FYI, personally I'd prefer to stick with "resolved", which also is the term usually used by ticket / issue systems.

Reply:
Same here, it seems odd to use "resolution" in the configuration and call it "addressed" here. Also I'm not sure if "addressed" has the right meaning, from a dictionary:
* "address": think about and begin to deal with (an issue or problem)
* "resolve": settle or find a solution to (a problem or contentious matter)

I think the latter matches better what error resolutions are there for.

Reply:
This is what dictionaries say. However, if I say, "I've addressed a problem", the common understanding is that I've dealt with it and it no longer needs attention. Technically you could regard 'resolve' as being more precise, because it says that a resolution to the problem has been found and applied, which of course does not cover problems that turn out not to require a solution, but are listed as problems until someone has 'addressed' them. If you want to avoid changing some existing sense or usage, I wouldn't argue.